cul.js

```diff
@@ -136,7 +141,8 @@ var Cul = function (options) {
 
     this.write = function send(data, callback) {
         //console.log('->', data)
-        serialPort.write(data + '\r\n', callback);
+        serialPort.write(data + '\r\n');
+        serialPort.drain(callback);
```

Comment:
Can confirm this. Had the same issue with the Callback (Raspberry Pi 3 / Busware CC1101-USB-Lite). With these changes it is working perfectly.